server/db/interface.go

```diff
@@ -3,6 +3,32 @@
 
 package db
 
+// TODO
+//
+// Epochs:
+//  1. PK: epoch ID
+//  2. list of order IDs and order hashes (ref orders table)
+//  3. resulting shuffle order
+//  4. resulting matches (ref matches table)
+//  5. resulting order mods (change filled amount)
+//  6. resulting book mods (insert, remove)
+//  refs: orders, matches
```

Comment:
I'm still uncertain about storing epoch related data.  It's probably helpful for auditing, but I don't know if it is necessary beyond just the matches and orders in their own tables.  If the matches contain an epoch identifier, then the epoch ordering can be recomputed easily.

Reply:
We can add it later if we need it. It's not a necessity.